.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .discourse-site
-HELP
-./bootstrap
```

Comment:
@duranmla what's this?

Reply:
HELP is a file that comes with the default creation of a community theme using CLI https://meta.discourse.org/t/discourse-theme-cli-console-app-to-help-you-build-themes/82950 the `./bootstrap` was one that I added before because I though that maybe we will have a bootstrap folder to get the snippets but actually that `./bootstrap` is not correct and neither used.